### ISSUE TITLE
better explicitly organize what are the team and commonjs plugins in the list. have distinction

### ISSUE THREAD

**Issue by ghost** [stylelint/awesome-stylelint#60]
### Describe the documentation issue

Yeah in the documentation lists, I would like to see a difference between CommonJS plugins and ESM plugins, because right now I can't know what plugin I can install!!

### What solution would you like to see?

I would like to see more description in the readme file. Let's be alive and kicking , better organization, better life
Also,

**Comment by ybiquitous** [comment]
I'm closing this due to lack of information.

**Comment by romanmenke** [comment]
Now that Stylelint 16 is released, some plugins are compatible with it and still work as expected. However, some plugins still do not support Stylelint 16, and just work as expected until Stylelint 15.

The current issue is suggesting that: the main README.md file in this repository could list the plugins available in a more precise way.
Instead of listing all the plugins available for Stylelint,    you could make it clear what are those plugins compatible with Stylelint 16 in the list of plugins, and what are the plugins that were developed until Stylelint 15.

In this case we need this distinction, since plugins designed taking CommonJS in mind won't ever work with Modules/Stylelint 16.
The README.md is giving a wrong impression that all those plugins are going to work with Stylelint 16, and they won't. Only a few plugins are working with Stylelint 16.
Please reconsider opening this issue.
@ybiquitous

**Comment by ybiquitous** [comment]
I can't entirely agree with describing supported Stylelint versions because we would need to take more time for the maintenance work to check supported versions.

**Comment by jeddy3** [comment]
> plugins designed taking CommonJS in mind won't ever work with Modules/Stylelint 16

Stylelint 16.0.0 supports both CommonJS and ESM plugins as it's a hybrid package. We only deprecated the CommonJS Node.js API in 16.0.0 and now show a deprecation warning for plugins that use `.cjs`.

> Only a few plugins are working with Stylelint 16

I was under the impression that only one or two plugins weren't working (because they had misconfigured exports)

Can you help us by drawing up a more definitive list of plugins that aren't working so we know where we stand?

**Comment by maria-agree** [comment]
Great way of handling feedback, you two. Thanks.

As time goes by, some plugins that weren't near of being updated, have now gotten updated. Most of plugins that I install are working now. I only see a problem remaining with the selector-bem-pattern plugin at the moment. However, since I made other changes in the stack besides updating Stylelint to version 16, I can't say that the error I see is happening on this plugin due to Stylelint 16.0.0 update.

I did not envision properly that this task would increase maintenance. Yeah, it is difficult to see a solution that would not require a constant review of the situation. Maybe only an automatic script, and even then.

Sorry for taking up some time. This may have been pertinent in november 2023, but not anymore now in february 2024.
